norfair/tracker.py

```diff
@@ -746,7 +746,7 @@ class Detection:
     Parameters
     ----------
     points : np.ndarray
-        Points detected. Must be a rank 2 array with shape `(n_points, n_dimensions)` where n_dimensions is 2 or 3.
+        Points detected. Must be a rank 2 array with shape `(n_points, n_dimensions)`.
```

Comment:
did this change?

Reply:
The thing is that it is not actually a restriction to have dimension 2 or 3. You can have any dimension you want. I decided to change that description since I was already modifying the Detection class anyway